### PR TITLE
Rename commands and update references

### PR DESCRIPTION
**Pull request by chvck**
This is dependent on the config_rename PR going in first.

### CODE REVIEW

File: src/cli_options.rs [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+    Value,
+};
+use std::io::Write;
+
+#[derive(Clone, Debug)]
+pub struct CliOptions {
+    pub conn_string: Option<String>,
+    pub username: Option<String>,
+    pub password: bool,
+    pub cluster: Option<String>,
```
Comment (ingenthr) [inline src/cli_options.rs:17]: so, I guess these stay the same in the move to database from cluster?

**Review by ingenthr — APPROVED** [review]
LGTM.  Made a note on something to look at, but it could be handled in the future if required.

File: src/cli_options.rs (hunk 1 above)
Comment (chvck) [reply src/cli_options.rs:17]: These are just struct members used internally. Parsing into them happens below. You're right that cluster need renaming to database in the parsing too though.

Comment (chvck) [reply src/cli_options.rs:17]: Filed #229

**Pull request merged**